Compatibility/iMac14,1.json

```
@@ -0,0 +1,18 @@
+{
+    "MacName": "iMac (21.5-inch, Late 2013)",
+    "Author": "@MarkVDW",
+    "Details": "Performance is great if you have an SSD upgrade (but can't comment on HDD). Everything seems to work out of the box, 
+    no need to patch any kexts in the post install app. Trying to run the kext patching step will give errors - just skip the step.
+    Continuity works completely. Did not test FileVault or WPA2 Enterprise.",
```

```suggestion
    "Details": "Performance is great if you have an SSD upgrade (fusion drives run extremely slow). Everything seems to work out of the box, no need to patch any kexts in the post install app. Trying to run the kext patching step will give errors - just skip the step. Continuity works completely.",
```

Try to keep strings one line in JSON. Patched Sur will automatically move the text to separate lines. I also replaced your lack of comment on HDD with a comment on fusion drives because that is (probably) the reason why these Macs are not supported by Big Sur.